app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,11 @@
   </head>
 
   <body>
+  <%= render "shared/navbar" %>
+  <p class="notice"><%= notice %></p>
+  <p class="alert"><%= alert %></p>
+
     <%= yield %>
+    <%= render "shared/flashes" %>
```

Comment:
What does this refers to?

Reply:
This is the styling of devise notices of logins. I think.